app/views/languages/repository.erb

```diff
@@ -0,0 +1,70 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Exercism Repositories</h1>
+
+    <p class="lead">
+      <strong>What are the repositories you can contribute to?</strong><br />
+    </p>
+
+    <p>
+
+     Exercism is a project comprised of several repositories and these can be divided basically into:</p>
+
+      <h2> Basic components </h2>
+
+      <ul>
+        <li><strong><a href="https://github.com/exercism/exercism.io">The website</a></strong>: Holds the code that people submit. Browse and review solutions, and have conversations.</li>
+        <li><strong><a href="https://github.com/exercism/cli">Command line interface</a></strong>: Used to fetch exercises and submit solutions.</li>
+         <li><strong><a href="https://github.com/exercism/x-api">Problems API</a></strong>: Delivers exercises. Consumed by the Exercism command-line client, as well as the Exercism website.</li>
+           
+
+      </ul>
+
+       <h2> Interesting Tools </h2>
+
+      <ul>
+     
+       
+        <li><strong><a href="https://github.com/exercism/rikki">Rikki- the robot</a></strong>: Provides helpful, friendly, automated feedback on certain exercises.</li>
+        <li><strong><a href="https://github.com/exercism/blazon">Blazon</a></strong>: Helps us manage cross-track issues.</li>
+      </ul>
+    
+    </p>
+  </section>
+   <h2>Languages you can contribute to!</h2>
+
+  <hr>
+   
+    <% active.each_slice(6) do |tracks| %>
+      <div class="row">
+        <% tracks.each do |t| %>
+          <%= erb :'languages/_repo', locals: { track: t } %>
+        <% end %>
+      </div>
+    <% end %>
+
+    <section class="page-header">
+      <h3>Upcoming</h3>
+    </section>
+    <% inactive.each_slice(6) do |tracks| %>
+      <div class="row">
+        <% tracks.each do |t| %>
+          <%= erb :'languages/_repo', locals: { track: t }  if t.problems.any? %>
+        <% end %>
+      </div>
+    <% end %>
+
+    <section class="page-header">
+      <h3>Planned</h3>
+    </section>
```

Comment:
Let's add some text here to say something like... here are some languages that people have asked for, but that have not gotten much attention yet, and that if they're excited about having that language on Exercism they should pitch in.